MAINTAINERS.md

```diff
@@ -0,0 +1,21 @@
+# Armada Maintainers
+
+## Current
+
+| Maintainer           | GitHub ID                                               | Affiliation |
+| -------------------- | ------------------------------------------------------- | ----------- |
+| Carlo Camurri        | [carlocamurri](https://github.com/carlocamurri)         | G-Research  |
+| Sam Clark            | [samclark](https://github.com/samclark)                 | G-Research  |
+| Dave Gantenbein      | [dave-gantenbein](https://github.com/dave-gantenbein)   | G-Research  |
+| Kevin Hannon         | [kannon92](https://github.com/kannon92)                 | G-Research  |
+| Clifton Houck        | [ClifHouck](https://github.com/ClifHouck)               | G-Research  |
+| Chris Martin         | [d80tb7](https://github.com/d80tb7)                     | G-Research  |
+| James Murkin         | [JamesMurkin](https://github.com/JamesMurkin)           | G-Research  |
+| Dejan Zele Pejchev   | [dejanzele](https://github.com/dejanzele)               | G-Research  |
+| Jamie Poole          | [jimbobby5](https://github.com/jimbobby5)               | G-Research  |
+| Alex Scammon         | [stackedsax](https://github.com/stackedsax)             | G-Research  |
+| Rich Scott           | [richscott](https://github.com/richscott)               | G-Research  |
+| Albin Severinson     | [severinson](https://github.com/severinson)             | G-Research  |
+| Daniel Rastelli      | [theAntiYeti](https://github.com/theAntiYeti)           | G-Research  |
```

Comment:
It looks like this is sorted alphabetically by last name - should Daniel Rastelli be after Jamie Poole?

Reply:
Doh

Reply:
This is why my kids got their mother's last name (starts with P, middle of such lists) instead of mine (Wilson, always last) :-)